promql/promqltest/testdata/at_modifier.test

```diff
@@ -91,7 +91,8 @@ eval instant at 25s metric{job="1"} @ 50 + metric{job="1"} @ 100
   {job="1"} 15
 
 # Note that this triggers an info annotation because we are rate'ing a metric that does not end in `_total`.
-eval_info instant at 25s rate(metric{job="1"}[100s] @ 100) + label_replace(rate(metric{job="2"}[123s] @ 200), "job", "1", "", "")
+eval instant at 25s rate(metric{job="1"}[100s] @ 100) + label_replace(rate(metric{job="2"}[123s] @ 200), "job", "1", "", "")
+  expect info
```

Comment:
I would argue that we don't really care about the `info` annotation here. In fact, we only added the comment and the `eval_info` because the old `eval` always implied that there must be no annotations whatsoever. I would remove the comment in line 93 and nat add the `expect info`.